examples/mapper_quickstart.ipynb

```diff
@@ -543,6 +542,52 @@
     "# Generate interactive widget\n",
     "plot_interactive_mapper_graph(pipe, data, color_data=data)"
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "  2. (**recommended**, new in ``giotto-tda`` 0.5.0) in an object-oriented way, by instantiating a ``MapperInteractivePlotter`` onject and then calling its ``plot`` method to create the widget."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Create the plotter object\n",
+    "MIP = MapperInteractivePlotter(pipe, data)\n",
+    "\n",
+    "# Generate interactive widget\n",
+    "MIP.plot(color_data=data)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The advantage of using the class API with ``MapperInteractivePlotter`` is that, once you are done tweaking the hyperparameters, you can inspect the latest state of the objects (graph, colours, pipeline, inner figure) which got changed during the interactive session."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "print(\"Attributes updated during plotting:\\n\",\n",
+    "      [attr for attr in dir(MIP) if attr.endswith(\"_\") and attr[0] != \"_\"])"
```

Comment:
How do we know that these are exactly those which changed?

Reply:
These are the attributes which are created only when calling .plot.  And they are also updated if we interact.  I know what you mean, but can't find a way to say this that isn't too long. Suggestions?

Reply:
I was thinking about something in line of ``Attributes that led to the current state of the graph ``. This cell is not a deal-breaker :)

Reply:
How about "Attributes created by `.plot` and updated during the interactive session"?

Reply:
Yes!

Reply:
Done.